Xamarin.Forms.Platform.Android/Renderers/FloatingActionButtonRenderer.cs

```diff
@@ -0,0 +1,337 @@
+﻿using Android.Content;
+using Android.Content.Res;
+using Android.Support.V4.View;
+using Android.Views;
+using System;
+using System.ComponentModel;
+using Xamarin.Forms.Internals;
+using AView = Android.Views.View;
+using AFloatingButton = global::Android.Support.Design.Widget.FloatingActionButton;
+using Xamarin.Forms.Platform.Android.FastRenderers;
+using Android.Graphics.Drawables;
+using Android.Widget;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public class FloatingActionButtonRenderer : AFloatingButton,
+		IVisualElementRenderer, IViewRenderer, ITabStop,
+		AView.IOnAttachStateChangeListener, AView.IOnFocusChangeListener, AView.IOnClickListener, AView.IOnTouchListener
+	{
+		const float SmallSize = 44;
```

Comment:
We have these constants in each renderer with the same values. I would move them to the abstraction.

Reply:
I have moved these const values directly to the enumeration that defines the sizes.